articles/aks/faq.md

```diff
@@ -177,6 +177,16 @@ Most clusters are deleted upon user request; in some cases, especially where cus
 
 No, you're unable to restore your cluster after deleting it. When you delete your cluster, the associated resource group and all its resources will also be deleted. If you want to keep any of your resources, move them to another resource group before deleting your cluster. If you have the **Owner** or **User Access Administrator** built-in role, you can lock Azure resources to protect them from accidental deletions and modifications. For more information, see [Lock your resources to protect your infrastructure][lock-azure-resources].
 
+## What is platform support, and what does it include?
+
+Platform support is a reduced support plan for unsupported "N-3" version clusters. Platform support does not include cluster or node pool creation, hotfixes, bug fixes, security patches, retired components, Kubernetes components, etc. See [platform support policy][supported-kubernetes-versions] for additional restrictions.
```

Comment:
We should have the general summary of what it does include right? before the link? Only Azure infrastructure support or something